addons/web/static/src/legacy/js/views/search_panel_model_extension.js

```diff
@@ -783,6 +783,6 @@
     }
     SearchPanelModelExtension.layer = 1;
 
-    ActionModel.registry.add("SearchPanel", SearchPanelModelExtension, 30);
+    ActionModel.registry.add("search_panel", SearchPanelModelExtension, 30);
```

Comment:
camel case here?

Reply:
why? But actually I'm not sure I would have changed all the "Lunch" -> "lunch" or "Forecast" -> "forecast". For now, it doesn't matter if in the sharedState the keys don't follow the same naming convention. In these cases, they are only accessed from the legacy code anyway.

Reply:
let us camel case
don't change lunch,...